BookingsApi/Testing.Common/Configuration/Config.cs

```diff
@@ -7,6 +8,7 @@ public class Config
         public AzureAdConfiguration AzureAdConfiguration { get; set; }
```

Comment:
Must the concrete version be used for these declarations (do any of these have an interface that could be substituted in)?

Reply:
This class is duplicated in all the projects. :(